README.md

```diff
@@ -5,4 +5,38 @@
 [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=com.powsybl%3Apowsybl-gse&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.powsybl%3Apowsybl-gse)
 [![MPL-2.0 License](https://img.shields.io/badge/license-MPL_2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
 
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
```

Comment:
I think you really have to add the last sequence of the site introduction : "Its modular approach allows developers to extend or customize its features." Modular and easy customized are two strong characteristics of powsybl.